static/styles/uikit/custom/components.scss

```diff
@@ -2,14 +2,104 @@
   color: $global-primary-background;
   font-weight: bold;
   text-transform: uppercase;
-}
 
-.logo__trailing {
-  font-weight: normal;
-  text-transform: capitalize;
+  &--inverse {
+    color: $global-muted-color;
+  }
+
+  &:not(.logo--inverse) .logo__trailing {
+    font-weight: normal;
+    text-transform: capitalize;
+  }
```

Comment:
Remover o :not, pois o texto Finance precisa estar com o font-weight normal e text-transform capitalize em todas as variação da logo

Reply:
Resolvido no commit accb3ba38e883afdb7116ba83b1dd02135cc974d